social_media_icon_extention_xing/README.md

```diff
@@ -0,0 +1,63 @@
+
+<center>
+<a href="http://www.xing.com/">
+<img class="oe_picture" width="100" src="https://github.com/blooparksystems/website/blob/8.0/website_social_media_icon_extention/static/description/XING.png?raw=true">
+</a>
+
+Extend Social Media Links
+=========================
+
+<img class="oe_picture" width="700" src="https://github.com/blooparksystems/website/blob/8.0/website_social_media_icon_extention/static/description/settings.png?raw=true">
+<img class="oe_picture" width="700" src="https://github.com/blooparksystems/website/blob/8.0/website_social_media_icon_extention/static/description/webpage.png?raw=true">
+
+
+Currently added:
+
+Xing
+
+</center>
+
+
+Integration example:
```

Comment:
Mention also that by default, icon is shown on the sections where it appears (footer, about us...)